### ISSUE TITLE
LESS mixins: Autoprefixer inserts curly braces {} after mixin with braces

### ISSUE THREAD

**Issue by qbasic16** [mrmlnc/vscode-autoprefixer#8]
As the title says, the problem is with LESS files, that if you use mixis with braces like mentioned on the official page, then autoprefixer replaces the last part "();" with "{}". Note that the trailing semicolon is removed as well.
[LESS - Mixin feature](http://lesscss.org/features/#mixins-feature)

Before autoprefixer:
```less
.slick-btns{
	.absolute();
	.p-top-right();
}
```
After autoprefixer:
```less
.slick-btns{
	.absolute(){}
	.p-top-right(){}
}
```

LESS will not insert the mixins and therefore produces an empty .slick-btns class in my example.

Thanks for your work.

Best regards
Peter

**Comment by mrmlnc** [comment]
Hello, @qbasic16,

Thanks for you report. Unfortunately, this is not a problem of this extension. To work with Less Autoprefixer uses [postcss-less](https://github.com/webschik/postcss-less). I copy your question in their repository and as soon as the bug is fixed, I'll update this plugin.

**Comment by qbasic16** [comment]
Hi @mrmlnc 

Thank you for your extremely fast response and for reporting it!
I subscribed to it.

**Comment by mrmlnc** [comment]
I reopen your issue to not forget about it.

**Comment by mrmlnc** [comment]
Available in v1.1.0
